controllers/nf/amf/reconciler.go

```diff
@@ -172,7 +172,7 @@ func (r *AMFDeploymentReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 			}
 		}
 	} else {
-		log.Error(err, fmt.Sprintf("Failed to create Deployment %s\n", err.Error()))
+		log.Error(err, fmt.Sprintf("Failed to create Deployment %s", err.Error()))
```

Comment:
```suggestion
		log.Error(err, fmt.Sprintf("Failed to create AMF NFDeployment %s", err.Error()))
```

Reply:
Actually, this log error is about failure to creating a K8s Deployment resource, I will keep the original.